proto/service.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+package main;
+option go_package = "github.com/SandyWalsh/runner/gen";
+
+message AuthRequest {
+    string username = 1;
+}
+
+message LaunchRequest {
+    string sessionID = 1;
+    string cmd = 2;
+    repeated string arg = 3;
+}
+
+message NoArg {};
+
+message Status {
+    enum StatusEnum {
+        ERROR = 0;
+        RUNNING = 1;
+        COMPLETED = 2;
```

Comment:
Can we set this up so that it's possible to identify whether a job completed naturally or was stopped via the `Abort` RPC?

Reply:
sounds reasonable.